jupyterlite_xeus/add_on.py

```diff
@@ -300,14 +298,16 @@ def copy_kernel(self, kernel_dir, kernel_wasm, kernel_js, kernel_data):
             ],
         )
 
-        yield from self.pack_prefix(kernel_dir=kernel_dir)
+        # pack prefix packages
+        # TODO Prevent duplication of the env pack for each kernel in it
```

Comment:
I believe we should track that in an issue. Also if the same kernel is installed in two envs it will not work well, let's also track that in an issue.